Files/Views/LayoutModes/GridViewBrowser.xaml.cs

```diff
@@ -471,5 +471,20 @@ private void FileList_DoubleTapped(object sender, DoubleTappedRoutedEventArgs e)
                 ParentShellPageInstance.InteractionOperations.OpenSelectedItems(false);
             }
         }
+
+        public async void PinItemToStart_Click(object sender, RoutedEventArgs e)
+        {
+            foreach (ListedItem listedItem in SelectedItems)
+            {
+                listedItem.IsItemPinnedToStart = await App.SecondaryTileHelper.TryPinFolderAsync(listedItem.ItemPath, listedItem.ItemName);
```

Comment:
With the change above can become `await App.SecondaryTileHelper.TryPinFolderAsync(listedItem.ItemPath, listedItem.ItemName);`